CMakeLists.txt

```diff
@@ -139,7 +150,7 @@ if(NOT LAPACK_FOUND)
     endif()
 endif()
 
-set(SPDLOG_BUILD_TESTING OFF)
+set(SPDLOG_BUILD_TESTING OFF CACHE INTERNAL "Disable testing in spdlog")
```

Comment:
This is only preventing spdlog-utests from being added into our terst with latest CMake versions, perhaps a bug, CMake versions older than 3.12 or 3.13 still have this problem - I faced this issue while working on github actions.